### ISSUE TITLE
Auth0 Electron Callback URL mismatch

### ISSUE THREAD

**Issue by Toosick** [auth0-samples/auth0-electron-samples#25]
@chenkie @auth0 I can't seem to get `file:///` to work with our Electron app. I was able to get things working by adding the full callback `file:///` url into the `Allowed Callback URLs` setting. The callbacks work just fine on the web using the callback with `https://*.netlify.com/login`. Does it now have to be something like `file://*` or `file:///*`?

Client ID: QR579kSOEqhOqPAfqjfTPHaz5hJPvEKT
Auth0 lock version: 11.0.1

### Callback mismatch
![Callback mismatch](https://image.prntscr.com/image/oWZ5OXuXQxaVU20Ugck_yQ.png "Callback mismatch")

### Auth0 lock setup
![Auth0 lock setup](https://image.prntscr.com/image/up0m50tWRxyuLXOfTaC00w.png "Auth0 lock setup")

**Comment by bkniffler** [comment]
Yes, I've tried to get electron running with auth0 and I've been running into the same problem.

**Comment by bkniffler** [comment]
Discussion happening here: https://github.com/auth0/lock/issues/1217
